src/utils/websockets.js

```diff
@@ -38,11 +40,21 @@ export function connectWS(topic, onMessageCallback) {
   };
 
   const connect = () => {
+    let reconnectCounter = 0;
     this.sockClient = new Client({
       brokerURL: config.WS_URL,
-      // debug: function(str) {
-      //   console.log('debug: ', str);
-      // },
+      debug(str) {
```

Comment:
what is that `str`? pls use a better name.

Reply:
(see https://github.com/ryanmcdermott/clean-code-javascript#variables)

Reply:
Done used strMessage